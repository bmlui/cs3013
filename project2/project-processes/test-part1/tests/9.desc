evaluate echo with no arguments
