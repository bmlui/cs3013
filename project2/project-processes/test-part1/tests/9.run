PATH="/bad" ./wshell < tests/9.in
