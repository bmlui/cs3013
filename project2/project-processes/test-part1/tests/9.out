test-part1$ echo

test-part1$ exit
